client/src/utils/RangeField/RangeField.tsx

```
@@ -0,0 +1,50 @@
+const RangeField: React.FC<RangeFieldProps> = ({
+  label,
+  minName,
+  maxName,
+  minValue,
+  maxValue,
+  onChange,
+  placeholder = { min: 'Min', max: 'Max' },
+}) => {
+  return (
+    <div className="mb-4">
+      <label className="block mb-2">{label}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)